### ISSUE TITLE
Data for United Kingdom wrong from Mar 16

### ISSUE THREAD

**Issue by alfkoehn** [CSSEGISandData/COVID-19#823]
It looks like we have another date with wrong data, **United Kingdom** has no increase according to this data, but the actual source is telling us a different story:

Total cases: 1391
Total deaths: 35
Source: https://www.arcgis.com/apps/opsdashboard/index.html#/f94c3c90da5b4e9f9a0b19484dd4bb14

Anybody else found some wrong data?

**Comment by alfkoehn** [comment]
**France** also seems to be wrong:

Total cases: 5423
Total deaths: 127
Source: https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde

**Comment by jagsfan82** [comment]
This seems to be just an issue with how often they are updating this github.  Data on their dashboard is more up to date.  Upsetting becuase those numbers were updated shortly after they published here.  Hopefully they push another update out early tomorrow (today) to fix these.  In general would be nice to see this updated every hour or so.. but not sure how involved it is for them to take their data on their dashboard and update here as well?  Or maybe theres another reason they don't want to update more often?  Less competition and more traffic to their dashboard?  Idk.

**Comment by bbstrikesagain** [comment]
Yes, 15th UK data not updated, deaths should be 35 for 15th, up from 21 for 14th.  In fact several previous rises in deaths are still missing so the data stalled on 8 instead of passsing through 10/11.

BUT, the problem seems to be in the WHO situation reports, they seem to be the source of the error., also not keeping up with the daily trend.  This is scary now, since reliable deaths reporting iwould be the only measure in the UK, because since 11th March the UK gov decided not to test mild cases, and to tell folk to stay at home and sweat it out for just seven days

**Comment by ginestrab** [comment]
I believe the UK deaths data is not only wrong for the 15th of March but also for other previous dates. As I remember there has been no date without an increases in deaths in the last week or so. The goverment might be reliasing the data slowing and not in time (the say they will release them at 2pm in the GOV.UK datasets but they are typically minutes if not hours late). The full time series needs to be double checked as this can perturbe the analysis of the data in UK.

**Comment by 3verse** [comment]
> Yes, 15th UK data not updated, deaths should be 35 for 15th, up from 21 for 14th. In fact several previous rises in deaths are still missing so the data stalled on 8 instead of passsing through 10/11.
> 
> BUT, the problem seems to be in the WHO situation reports, they seem to be the source of the error., also not keeping up with the daily trend. This is scary now, since reliable deaths reporting iwould be the only measure in the UK, because since 11th March the UK gov decided not to test mild cases, and to tell folk to stay at home and sweat it out for just seven days.
> 
> Similar problem with confirmed cases not being updated, but this may be moot point as non-test policy kicks in

true... I'm one of those 'sweating it out' as we speak, and I believe I'm being responsible about it - I have seen already several folks coughing heavily on my commute last week, and that's probably what's keeping me home now in the first place. To top it up I can't be tested so I'm not going to know if I'm affected or not in the first place unless I get worse and that will probably mean it will take a lot longer to recover... sorry, I know it's not relevant to the data, just venting for once. Stay safe!

**Comment by ginestrab** [comment]
I noticed that today 16/3/2020 the UK government has stopped relasing the data on the number of deaths. I urge John Hopkins University to make pressure to have reliable data for all the world-wide scientific community. Repetitions and missing data can effectively falsify results.

**Comment by przemyslawpluta** [comment]
**55** as of today but they've stopped releasing the data on the website [gov.uk](https://www.gov.uk/guidance/coronavirus-covid-19-information-for-the-public) timeseries is also incorrect https://github.com/CSSEGISandData/COVID-19/issues/714

**Comment by valeriupredoi** [comment]
see comment https://github.com/CSSEGISandData/COVID-19/issues/911#issuecomment-602223995 for an explanation why the deaths are wrong and how to retrieve the correct ones daily. Cheers :beer:
